<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DbConnectionDialog</class>
 <widget class="QDialog" name="DbConnectionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>273</width>
    <height>175</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DB Connection</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="hostNameLabel">
       <property name="text">
        <string>Host name:</string>
       </property>
       <property name="buddy">
        <cstring>hostNameLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="hostNameLineEdit">
       <property name="text">
        <string>localhost</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="dbNameLabel">
       <property name="text">
        <string>DB name:</string>
       </property>
       <property name="buddy">
        <cstring>dbNameLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="dbNameLineEdit">
       <property name="text">
        <string>LaboratoryAlgorithmDatabaseSchema</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="userNameLabel">
       <property name="text">
        <string>User name:</string>
       </property>
       <property name="buddy">
        <cstring>userNameLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="userNameLineEdit">
       <property name="text">
        <string>root</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="passwordLabel">
       <property name="text">
        <string>Password:</string>
       </property>
       <property name="buddy">
        <cstring>passwordLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="passwordLineEdit">
       <property name="text">
        <string>root</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="portLabel">
       <property name="text">
        <string>Port:</string>
       </property>
       <property name="buddy">
        <cstring>portLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="portLineEdit">
       <property name="text">
        <string>3306</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="connectPushButton">
       <property name="text">
        <string>&amp;Connect</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="quitPushButton">
       <property name="text">
        <string>&amp;Quit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
